src/test_main.py

```diff
@@ -0,0 +1,82 @@
+from hand_gesture_recognition import HandClassifier
+from inference import HandDetector
+from hand_gesture_recognition import HandClassifier
+import argparse
+import numpy as np
+import cv2
+
+
+def get_hand_from_img(image, boxes, enlargebox_px):
+    hands = []
+
+    for box in boxes:
+        x = box[0]
+        y = box[1]
+        w = box[2]
+        h = box[3]
+        # x, y, w, h = box
+        # enlarge box a bit
+        x -= enlargebox_px
+        y -= enlargebox_px
+        w += enlargebox_px * 2
+        h += enlargebox_px * 2
+
+        bottom_right_x = x+w
+        bottom_right_y = y+h
+
+        if x < 0:
+            x = 0
+        if y < 0:
+            y = 0
+        if bottom_right_x > image.shape[1]:
+            bottom_right_x = image.shape[1]
+        if bottom_right_y > image.shape[0]:
+            bottom_right_y = image.shape[0]
+
+        hand = image[y:bottom_right_y, x:bottom_right_x]
+        hands.append(hand)
+
+    return hands
+
+
+def main():
```

Comment:
Delete rest of the test_main.py

Reply:
I think it's really useful to have some easy to use example for fast developing. It's not ready yet solution and for example @Krisisss  or @Skwarson96  have to spend lass time on evaluating this for testing. In final stage of development we could remove  it or leave as "easy to use example"

Reply:
Ok